pkg/sosreport/index.html

```diff
@@ -28,20 +28,124 @@
   </head>
   <body class="pf-m-redhat-font" hidden>
     <div class="pf-c-page">
-      <main class="pf-c-page__main" tabindex="-1">
-        <section class="pf-c-page__main-section pf-m-light">
-          <div class="pf-c-empty-state">
-            <div class="pf-c-empty-state__content">
-              <img class="pf-c-empty-state__icon" aria-hidden="true" src="sosreport.png" alt="">
-              <div class="pf-c-empty-state__body">
-                <p translate="yes">This tool will collect system configuration and diagnostic information from this system for use with diagnosing problems with the system.</p>
-                <p translate="yes">The collected information will be stored locally on the system.</p>
-                <p id="switch-instructions" translate="yes">You need to switch to "Administrative access" in order to create reports.</p>
+        <main class="pf-c-page__main" tabindex="-1">
+            <section class="pf-c-page__main-section pf-m-light">
+              <div class="pf-c-empty-state">
+                <div class="pf-c-empty-state__content">
+                  <img class="pf-c-empty-state__icon" aria-hidden="true" src="sosreport.png" alt="" style="width:250px;height:250px;">
+                  <div class="pf-c-empty-state__body">
+                    <p translate="yes">This tool will collect system configuration and diagnostic information from this system for use with diagnosing problems with the system.</p>
+                    <p translate="yes">The collected information will be stored locally on the system.</p>
+                    <p id="switch-instructions" translate="yes">You need to switch to "Administrative access" in order to create reports.</p>
+                </div>
+                <button id="create-button" translate="yes" class="pf-c-button pf-m-primary">Create report</button>
+              </div>
+            </section>
+        </main>
+    </div>
+
+    <div hidden id="sos-options-menu">
+        <div class="pf-c-backdrop">
+            <div class="pf-l-bullseye">
+                <div class="pf-c-modal-box pf-m-md pf-m-align-top" role="dialog" aria-modal="true">
+                    <button class="pf-c-button pf-m-plain" type="button" aria-label="Close" id="sos-options-close-button">
+                        <i class="fa fa-times" aria-hidden="true"></i>
+                    </button>
+                    <header class="pf-c-modal-box__header">
+                        <h1 class="pf-c-modal-box__title" translate="yes">Set SoS Report Options</h4>
+                        <small>All settings are optional, if in doubt leave default values and click 'Run Report'</small>
```

Comment:
I don't think we need to specify this. If we do, then maybe it should be behind (?) or even in documentation. I think it is rather clear that those options are optional.

Reply:
You need to mark all human readable strings as translatable. For tag contents, `translate="yes"`, for the input titles `translate="title"`.

(And ugh, yes, this really ought to be React..)

Reply:
Ack to both